src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnYubicoCredentialsRepository.java

```diff
@@ -41,7 +42,7 @@ public WebAuthnYubicoCredentialsRepository(String provider,
             for (WebAuthnCredential c : credentials) {
                 PublicKeyCredentialDescriptor descriptor = PublicKeyCredentialDescriptor.builder()
                         .id(ByteArray.fromBase64(c.getCredentialId())).type(PublicKeyCredentialType.PUBLIC_KEY)
-                        .transports(c.getTransports())
+                        .transports(getTransportsFromString(c.getTransports()))
```

Comment:
use StringUtils.collectionToCommaDelimitedString and StringUtils.commaDelimitedListToSet to transform to/from

Reply:
Fixed in 674587d6d918161c1fd98065e2cef0484a6f55da